src/services/ubi/beneficiary.ts

```diff
@@ -520,15 +520,15 @@ export default class BeneficiaryService {
                     as: 'user',
                 },
             ],
-            order: [['createdAt', 'DESC']],
+            order: [['txAt', 'DESC']],
             limit,
             offset,
         });
         return transactions.map((transaction) => ({
             id: transaction.id,
             type: 'transaction',
             tx: transaction.tx,
-            date: transaction.createdAt,
+            date: transaction.txAt,
```

Comment:
```suggestion
            txAt: transaction.txAt,
```
Since this is not implemented on the UI, can you please change it?